app/scripts/lib/multichain-api/scope/validation.ts

```diff
@@ -68,19 +68,19 @@ export const isValidScope = (
   }
 
   // Note we are not validating the chainId here, only the namespace
-  const areAccountsValid = (accounts || []).every((account) => {
-    try {
-      return parseCaipAccountId(account).chain.namespace === namespace;
-    } catch (e) {
-      // parsing caipAccountId failed
-      console.log(e);
-      return false;
-    }
-  });
+  // const areAccountsValid = (accounts || []).every((account) => {
+  //   try {
+  //     return parseCaipAccountId(account).chain.namespace === namespace;
+  //   } catch (e) {
+  //     // parsing caipAccountId failed
+  //     console.log(e);
+  //     return false;
+  //   }
+  // });
 
-  if (!areAccountsValid) {
-    return false;
-  }
+  // if (!areAccountsValid) {
+  //   return false;
+  // }
```

Comment:
we might need this to verify accounts from requestPermissions or the new popup are correct, not sure where that would go yet though. just commented it out for now